packages/admin-next/dashboard/src/hooks/api/api-keys.tsx

```diff
@@ -21,12 +19,17 @@ export const useApiKey = (
   id: string,
   query?: Record<string, any>,
   options?: Omit<
-    UseQueryOptions<AdminApiKeyResponse, Error, AdminApiKeyResponse, QueryKey>,
+    UseQueryOptions<
+      HttpTypes.AdminApiKeyResponse,
+      Error,
+      HttpTypes.AdminApiKeyResponse,
```

Comment:
q: shouldn't one of these be the allowed query params or `SelectParams`?

Reply:
Yeah, I think you are right. It looks like we haven't used this much across the hooks, so there might be some clean-up work to do. I'll change this one 👍